docs/tutorials/setting-up-python-pycharm.md

```diff
@@ -198,3 +185,84 @@ typing `m.` and pycharm will suggest all possible methods and properties.
 Choose one. Now you can put your cursor inside the property and press F1 to
 see the docs. Or press cmd-down arrow to look inside the library where this
 property is defined. Neat huh? Happy coding.
+
+## Additional features for the PyCharm Professional version
+
+The Professional version of PyCharm (which can be obtained for free for educational institutions and OSS projects) has a few more features to make your development experience much smoother. If you are a student or teacher, register your valid university email address on the [JetBrains student application page](https://www.jetbrains.com/student/) to apply for the free version of PyCharm professional. If you are an open source developer, you can visit the [JetBrains Open Source license page](https://www.jetbrains.com/buy/opensource/) to see if you qualify for a free version of PyCharm professional.
+
+Here is a list of the tools you can use with PyCharm professional to make your Python development easier.
+
+| Tool + link | Purpose |
+| ----------- | ------- |
+| [SFTP Remote Server][remote-server] | An alternative to the approach used above to transfer files from your host computer to the EV3 pretty instantly when the file is saved on your host computer. |
+| [SSH Terminal][pycharm-ssh] | Lets you make the SSH connection directly within PyCharm instead of using a separate tool (like PuTTY). |
+| [Remote Interpreter][remote-interpreter] | Let's you avoid needing an SSH terminal since you just hit play within PyCharm to run programs. Allows debugging, logs, etc. all within your host computer PyCharm IDE (but it can be SLOW!) |
+
+With these tools and installing ev3dev-lang-python on your host computer, you can really make the Python development process pretty user friendly.
+
+### Setting up an SFTP Remote Server
+
+This is an alternative to the git post-receive hook solution that was shown earlier in this tutorial. If you have access to the Professional version of PyCharm, then this solution is a bit more elegant than the solution shown earlier. It works about the same in that it instantly transfers files, but it also allows you to have some other git remote (say github) instead of using the ev3 as your git remote. Here are the steps necessary to setup your file transfer mechanism using a PyCharm Remote Server.
+- Within PyCharm choose `File --> Settings` for Windows and Linux (or `PyCharm --> Preferences` for OS X)
```

Comment:
Also, I don't know that we have a well-established convention on the www.ev3dev.org site, but for labels that you see on UI elements of a desktop application, I usually use *italics* rather than `code` formatting style. Also, for menus, *File > Settings* rather than `File --> Settings`.